src/AccessControl/Permissions.php

```diff
@@ -515,7 +515,7 @@ public function isAllowed($what, $user, $content = null, $contentId = null)
     {
         if (is_array($content)) {
             $contenttypeSlug = $content['slug'];
-        } elseif ($content instanceof \Bolt\Content) {
+        } elseif ($content instanceof \Bolt\Legacy\Content) {
```

Comment:
This whole thing is just a first step in transitioning this stuff so it isn't stuck in someone's branch